src/tests/Interop/Swift/Dummy.swift

```diff
@@ -0,0 +1 @@
+public func dummyFunc() { }
```

Comment:
This doesn't seem like a helpful description. It would be helpful to name these in a manner that describes what it is for or what it is, perhaps `voidVoidFunc` or `simpleFunc`?

Reply:
The `simpleFunc` sounds good as a placeholder for invalid scenarios.